index.js

```diff
@@ -0,0 +1,156 @@
+//task №1
+
+const arrayNumbers = [1 , 2, 3, 4, 5, 6, 7, 8];
+
+let convertNumbersInArray = (arrayNumbers) => {
+	return arrayNumbers.map((number) => (number % 2 === 0) ? number += 1 : number -= 1);
+}
+
+console.log(convertNumbersInArray(arrayNumbers));
+
+
+//task №2
+
+const arrayStrings = ['hello', 'wOrLd'];
+let convertStringsInArray = (arrayStrings) => {
+	return arrayStrings.map((string) => string[0].toUpperCase() + string.slice(1).toLowerCase());
+}
+
+console.log(convertStringsInArray(arrayStrings));
+
+
+//task №3
+
+const usersList = [
+	{
+		id: 1,
+		name: 'Alex',
+	},
+	{
+		id: 2,
+		name: 'Toma',
+	},
+	{
+		id: 3,
+		name: 'Max',
+	},
+]
+
+let showUsers = (usersList, id) => {
+	return usersList.filter((user) => {
+		return !id.includes(user.id);
+	});
+}
+
+console.log(showUsers(usersList, [1]));
+
+
+//task №4
+
+let forCheckPalindrome = (value) => value.split('').reverse().join('') === value;
+
+console.log(forCheckPalindrome('112211'));
+
+
+//task №5
+const users = [
+	{
+		id: 1,
+		first_name: 'Jeanette',
+		last_name: 'Penddreth',
+		email: 'jpenddreth0@census.gov',
+		gender: 'Female',
+		ip_address: '26.58.193.2',
+	},
+	{
+		id: 2,
+		first_name: 'Petr',
+		last_name: 'Jackson',
+		email: 'gfrediani1@senate.gov',
+		gender: 'Male',
+		ip_address: '229.179.4.212',
+	},
+];
+
+let convertUsersNewArray = (users) => {
+	return users.map((user) => {
+		return {
+			id: user.id,
+			fulName: `${user.first_name} ${user.last_name}`,
+			email: user.email,
+			gender: user.gender,
+			ip_address: user.ip_address,
+		}
+	});
+}
+
+let convertUsersNewArrayWomen = (users) => {
+	return users.filter((user) => user.gender.toLowerCase() === 'female');
+}
+
+let convertUsersNewArrayMan = (users) => {
+	return users.filter((user) => user.gender.toLowerCase() === 'male');
+}
```

Comment:
nice to see that you decided to separate this logic into another methods. 
But I see the following issues:
1. You didn't follow DRY principle. Don't Repeat Yourself.  You could add the second param, like `gender` and write this function only once, then call it with different geneders
2. Performance issue: You iterates through the single array twice, but this task could be done via single circle